From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Brandon Williams <turbine.bmw@gmail.com>
Date: Sun, 19 Jul 2026 18:46:35 -0400
Subject: [PATCH] dt-bindings: net: qcom,ath12k: document disable-rfkill

Allow boards without a usable hardware rfkill input to request that
the driver skip firmware rfkill configuration.

Signed-off-by: Brandon Williams <turbine.bmw@gmail.com>
---
 .../devicetree/bindings/net/wireless/qcom,ath12k.yaml       | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/wireless/qcom,ath12k.yaml b/Documentation/devicetree/bindings/net/wireless/qcom,ath12k.yaml
index dc68dd59988f..6de83f18b63b 100644
--- a/Documentation/devicetree/bindings/net/wireless/qcom,ath12k.yaml
+++ b/Documentation/devicetree/bindings/net/wireless/qcom,ath12k.yaml
@@ -27,6 +27,12 @@ properties:
       If present, a board or platform specific string used to lookup
       usecase-specific firmware files for the device.
 
+  disable-rfkill:
+    type: boolean
+    description:
+      Skip firmware rfkill configuration on platforms that do not expose a
+      usable hardware rfkill input to the wireless device.
+
   vddaon-supply:
     description: VDD_AON supply regulator handle
 
